AmericanMedicalAssociation/ama-style-guide-2
# ADF-128: Search filters

**Jira Ticket**
- [ADF-128 &127 : Prototype search filters (open and closed)] (https://palantir.atlassian.net/browse/ADF-128 ) &  (https://palantir.atlassian.net/browse/ADF-127)

## Description
* Create search filters

## Test

- Checkout the **ADF-128-search-filters** branch and cd into styleguide, run **gulp serve** to spin up the pattern lab style guide.

- Verify that **dr-finder-filter-list** folder and related files exist  under 
   styleguide/source/_patterns/02-molecules.

- Verify that **_dr-finder-filter-list.scss** file and **dr-finder-filter-list.js** exists under  styleguide/ source / assets

- Navigate to styleguide site to make sure the Dr finder filter list is function properly.

## Relevant Screenshots/GIFs
![Screen Shot 2022-10-04 at 12 14 24 PM](https://user-images.githubusercontent.com/54925022/193897238-1cf1cf19-92d2-4c0b-932e-f992e39afc31.png)
![Screen Shot 2022-10-04 at 12 14 50 PM](https://user-images.githubusercontent.com/54925022/193897263-31319d28-41ab-485f-9cb3-ef303818432e.png)
![Screen Shot 2022-10-04 at 12 14 57 PM](https://user-images.githubusercontent.com/54925022/193897283-3db9b99b-f170-43e3-8932-ce982659973c.png)
